cfgov/v1/blocks.py

```diff
@@ -0,0 +1,32 @@
+from wagtail.wagtailcore import blocks
+from v1.util import util
+
+
+class AbstractFormBlock(blocks.StructBlock):
+    """
+    Block class to be subclassed for blocks that involve form handling.
+    """
+    def get_result(self, page, request, value, is_submitted):
+        handler_class = self.get_handler_class()
+        handler = handler_class(page, request, block_value=value)
+        return handler.process(is_submitted)
+
+    def get_handler_class(self):
+        handler_path = self.meta.handler
+        if not handler_path:
+            raise AttributeError(
+                'You must set a handler attribute on the Meta class.')
+        return util.load_class(handler_path)
```

Comment:
Could you use the built-in [`django.utils.module_loading.import_string`](https://docs.djangoproject.com/en/1.8/ref/utils/#django.utils.module_loading.import_string) here instead?

```py
>>> from django.utils.module_loading import import_string
>>> LandingPage = import_string('v1.models.landing_page.LandingPage')
>>> LandingPage
<class 'v1.models.landing_page.LandingPage'>
```